Subsonicious/Subsonicious/Views/LoginView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  LoginView.swift
+//  Subsonicious
+//
+//  Created by Bilal on 15/06/2020.
+//  Copyright © 2020 Bilal Benlarbi. All rights reserved.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+
+    @EnvironmentObject var authentication: Authentication
+    @State var server = Server()
+
+    var body: some View {
+        GeometryReader { content in
+            VStack {
+                Text("login.login")
+
+                VStack {
+                    TextField("login.server", text: self.$server.address)
+                        .keyboardType(.URL)
+                        .textContentType(.URL)
```

Comment:
I think the contentType changes the keyboard type too, so you don't need the first line :)

Reply:
No, it doesn't